CMakeLists.txt

```diff
@@ -0,0 +1,572 @@
+# # CMake project for fdk-aac
+#
+# ## Basic usage
+#
+# 1. Download and install CMake >= 3.10 (CMake >= 3.15 is recommended)
+# 2. Clone fdk-aac repository
+# 3. In fdk-aac directory create and go to `build` subdirectory:
+#
+#    mkdir build
+#    cd build
+#
+# 4. Run cmake to configure project:
+#
+#    cmake ..
+#
+#    If you want optimized release build (default is debug), add CMAKE_BUILD_TYPE=Release option:
```

Comment:
Could you make release the default?

Reply:
I've fixed documentation instead of it. No `CMAKE_BUILD_TYPE` is special mode, which allows to set all compilers flags from command line.